tests/properties.json

```diff
@@ -52,6 +53,13 @@
       "protocol": "http",
       "version": ""
     },
+    "STH": {
+      "basiccheck": "/",
+      "heartbeat": "/",
+      "port": "8666",
```

Comment:
Right now without PEP: 
STH_PORT: The port where the STH server will be listening. Optional. Default value: 8666.
Via: [IoT-STH](https://github.com/gtorodelvalle/IoT-STH)